.travis.yml

```diff
@@ -41,10 +41,10 @@ jobs:
 
       addons:
         apt:
-          packages: [clang-format-9]
+          packages: [clang-format-10]
```

Comment:
How about we make the dependency explicit?

```suggestion
          packages: [clang-10, clang-format-10]
```

Reply:
No, this bit only does the formatting, it doesn't need the compiler.

Reply:
Ah sorry, clang-10 is installed in the other one. I suppose this is just for conformity.